### PR TITLE
Bump media-sound/teamspeak-client:5.0.0_beta70

### PR DESCRIPTION
**Pull request by TomJuri**
You also couldn't install the previous version, because the link would always download the latest beta, and then the hashs wouldn't match. That's fixed.

### CODE REVIEW

File: media-sound/teamspeak-client/Manifest [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
 DIST TeamSpeak3-Client-linux_x86-3.5.6.run 98637555 BLAKE2B 6f891e6846f8f899edb8ef107b7ea6518f413488525b7696863292d99d73614cae40631f5ed2341cce75ea62ffd94c60968ff6b5d587446d7469271835c01202 SHA512 dfc5ed3e2a92ed9f1cea2fdde5e40cf73e4a999ea782c434b26072ac0e7ac37d6305c06ae2d39c104eeee10639f325c7b96f538082e60224a68c0553fab0d12c
-DIST teamspeak-client-5.0.0_beta68.tar.gz 134468631 BLAKE2B ddc02ec9b5e9b8a3ea3fb629bd39ab4973b6e9cf610d90a37d1ebde08fc893ea087e18efa599eac497771ee31aa7f4e835d6e2b6a07fb536eee150d0747be9d7 SHA512 3c3061b1e42fa7a90ddcadde9d27d543529dea45e4da023bf0519588fc64ff5336f1fe0abdf4f9ab42f255959f766bc6fbfae9d6daa2e3d8b3ba1fc76b74b293
+DIST teamspeak-client-5.0.0_beta70.tar.gz 137244141 BLAKE2B f6b238de91773782e94158fba194d87a0229a058cef9d0065f1c6e32139c4e753772086d2186fc3522fbb6654daadaba77ac981abd643513a3586ebb35650443 SHA512 144ee2409a511ac46155db95a6f7b2d641052b34fb8cb2a26b0724a7b9aa7d718db392d2e32b460cf9ca0c379d56397759c2cb65c3ac74260fa45cfb27c5f740
```
Comment (ConiKost) [inline media-sound/teamspeak-client/Manifest:3]: Please squash Manifest update and ts client update into one commit.

File: media-sound/teamspeak-client/teamspeak-client-5.0.0_beta70.ebuild [hunk 2]
```diff
@@ -7,7 +7,7 @@ inherit desktop xdg-utils
 
 DESCRIPTION="A client software for quality voice communication via the internet"
 HOMEPAGE="https://www.teamspeak.com/"
-SRC_URI="https://files.teamspeak-services.com/teamspeak/teamspeak-client.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://files.teamspeak-services.com/pre_releases/client/5.0.0-beta70/teamspeak-client.tar.gz -> ${P}.tar.gz"
```
Comment (ConiKost) [inline media-sound/teamspeak-client/teamspeak-client-5.0.0_beta70.ebuild:10]: Please update ```5.0.0-beta70``` to variable usage, so version is not hard coded.

**Review by ConiKost — CHANGES_REQUESTED** [review]
Please also add a complete ```Sign-off-by``` to each commit.

File: media-sound/teamspeak-client/teamspeak-client-5.0.0-beta70.ebuild [hunk 3]
```diff
@@ -7,7 +7,7 @@ inherit desktop xdg-utils
 
 DESCRIPTION="A client software for quality voice communication via the internet"
 HOMEPAGE="https://www.teamspeak.com/"
-SRC_URI="https://files.teamspeak-services.com/teamspeak/teamspeak-client.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://files.teamspeak-services.com/pre_releases/client/${PV}/teamspeak-client.tar.gz -> ${P}.tar.gz"
```
Comment (ConiKost) [inline media-sound/teamspeak-client/teamspeak-client-5.0.0-beta70.ebuild:10]: This won't work, as ```PV=5.0.0-beta70```, but link was prior ```5.0.0_beta70```. So you need to adjust this, something like ```${PV/-/_}```, this should work.

Comment (TomJuri) [reply media-sound/teamspeak-client/teamspeak-client-5.0.0-beta70.ebuild:10]: The issue was with the previous name. This works:
https://files.teamspeak-services.com/pre_releases/client/5.0.0-beta70/teamspeak-client.tar.gz
This doesn't:

Comment (ConiKost) [reply media-sound/teamspeak-client/teamspeak-client-5.0.0-beta70.ebuild:10]: Ah! Thank you, so I did missunderstood. Then its fine of course.

Comment (ConiKost) [reply media-sound/teamspeak-client/teamspeak-client-5.0.0-beta70.ebuild:10]: Actually, no, this is not allowed. As per PMS, you need to use ```_beta``` with underscore as suffix. I haven't seen, that you changed the ebuild name here. The suffix ```-beta``` is not allowed.

Comment (TomJuri) [reply media-sound/teamspeak-client/teamspeak-client-5.0.0-beta70.ebuild]: Should be okay now?

**Review by ConiKost — CHANGES_REQUESTED** [review]
Yes, looks good. Please squash into one commit now.

File: media-sound/teamspeak-client/teamspeak-client-5.0.0_beta70.ebuild [hunk 4]
```diff
@@ -7,14 +7,14 @@ inherit desktop xdg-utils
...
-SRC_URI="https://files.teamspeak-services.com/teamspeak/teamspeak-client.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://files.teamspeak-services.com/pre_releases/client/${PV/-/_}/teamspeak-client.tar.gz -> ${P}.tar.gz"
 S="${WORKDIR}"
 
 KEYWORDS=""
 LICENSE="teamspeak5 || ( GPL-2 GPL-3 LGPL-3 )"
 SLOT="5"
 
-RDEPEND="
+RDEPEND="${PV}
```
Comment (ConiKost) [inline media-sound/teamspeak-client/teamspeak-client-5.0.0_beta70.ebuild:17]: Please remove ```${PV}``` from RDEPEND.

Comment (TomJuri) [reply media-sound/teamspeak-client/teamspeak-client-5.0.0_beta70.ebuild]: I swear I didn't do that

**Review by ConiKost — CHANGES_REQUESTED** [review]
Please squash all commits into one and add a proper ```Signed-off-by``` into your commit comment.

**Comment by TomJuri** [comment]
There we go

**Comment by TomJuri** [comment]
I'll just open another pull request lmao

**Pull request closed without merging**